convert2rhel/systeminfo.py

```diff
@@ -397,10 +413,58 @@ def corresponds_to_rhel_eus_release(self):
         For example if we detect CentOS Linux 8.4, the corresponding RHEL 8.4 is an EUS release, but if we detect
         CentOS Linux 8.5, the corresponding RHEL 8.5 is not an EUS release.
 
-        :return: Whether or not the current system has a EUS correspondent in RHEL.
+        :return: Whether or not the current system has an EUS correspondent in RHEL.
         :rtype: bool
         """
-        return "%s.%s" % (self.version.major, self.version.minor) in EUS_MINOR_VERSIONS
+        eus_release_date = EUS_MINOR_VERSIONS.get("%s.%s" % (self.version.major, self.version.minor), False)
```

Comment:
To simplify things, we could assign the default return value for this to be an empty string instead of a boolean value. 

```suggestion
        eus_release_date = EUS_MINOR_VERSIONS.get("%s.%s" % (self.version.major, self.version.minor), "")
```

Reply:
<nod> Note:  Both default values have use cases.  The way I decide is:

* If I just want to use the value later and my code has a fallback for when it wasn't set, then I'll use a value from within the data type that will work with that code (Empty string, empty list, and `False` are common defaults).  Example: My function gets a colon separated list of filenames from an environment variable, loops over each filename and performs an action.  I would use `for filename in os.environ.get("MYPATH", "").split(":"):` to retrieve the environment variable.  Empty string is perfect because it will split into an empty list which means the loop will not be executed.
 
* If there's a reason to differentiate between a default value and an explicitly set value, then I'll set it to `False` (or `None` if the variable holds a boolean).  Example: My function changes a number of attributes on a file, permissions, selinux context, owner, and group.  If the user has configured the permissions via a config value, then I want to use that.  If they have not and it's a new file, then I want to use a default value.  If they have not and it's an existing file, then I don't want to change the permissions at all.  So `config.get("mode", False)` is a better clue to the next reader of the code that we're dealing with a tri-state here.

This code is in the middle of those two criteria: If `eus_release_date` is not set, then it skips the rest of the code in the function and returns a hardcoded `False`, never using the value as a string.  In that circumstance, either style works here although my personal order of preference is probably:

* Use a try except:
  ``` python
  try:
      eus_release_date = 
        eus_release_date = EUS_MINOR_VERSIONS["%s.%s" % (self.version.major, self.version.minor)]
  except KeyError:
      return False
  ```
* Return False to be explicit that we're going to use this as a condition
* Use empty string.